Resources/Private/Templates/FrontEnd/FrontEnd.html

```diff
@@ -240,15 +240,14 @@
 <!-- ###SINGLE_VIEW###  -->
 
 <!-- ###FIELD_WRAPPER_TITLE### -->
-    <h2
-        <!-- ###FIELD_WRAPPER_IMAGE### -->
-            ###STYLE_SINGLEVIEWTITLE###
-        <!-- ###FIELD_WRAPPER_IMAGE### -->
-    >
+    <h2>
     <!-- ###FIELD_WRAPPER_EVENT_TYPE### -->
         ###EVENT_TYPE###:
     <!-- ###FIELD_WRAPPER_EVENT_TYPE### -->
     ###TITLE###</h2>
+    <!-- ###FIELD_WRAPPER_IMAGE### -->
+        ###STYLE_SINGLEVIEWTITLE###
```

Comment:
Please rename the marker as this isn't a style attribute anymore. I propose `SINGLE_VIEW_IMAGE`. What do you think?